config/common/common.go

```diff
@@ -110,8 +119,8 @@ func GetFullyQualifiedIDFn(serviceProvider string, keyPairs ...string) tjconfig.
 // rules first will be possible. Since there is no fixed index for key-value pairs in maps, it is not possible to place
 // rules from specific to general. Therefore, array is used here.
 var referenceRules = [][]string{
-	{"resource_group$", "/azure/v1alpha1.ResourceGroup"},
-	{"subnet$", "/network/v1alpha1.Subnet"},
+	{"resource_group$", ResourceGroupReferencePath},
```

Comment:
Please be careful here if you update the existing constant as it looks like we need the relative path here (no concatenation with the module name).